github/actions_oidc.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OIDCSubjectClaimCustomizationTemplate represents an OIDC subject claim customization template
+type OIDCSubjectClaimCustomizationTemplate struct {
+	UseDefault       *bool    `json:"use_default,omitempty"`
+	IncludeClaimKeys []string `json:"include_claim_keys"`
+}
+
+// GetOrganizationOIDCSubjectClaimCustomizationTemplate gets the subject claim customization template for an organization.
```

Comment:
Would it bother you if we shortened this to `GetOrgOIDCSubjectClaimCustomTemplate` ?

Reply:
Sounds good to me, I was looking at the other methods in the library and `Org` and `Organization` were used. Perhaps (as a future breaking change release), all the ones called `Organization` should be renamed to `Org` for consistency.

Reply:
> Sounds good to me, I was looking at the other methods in the library and `Org` and `Organization` were used. Perhaps (as a future breaking change release), all the ones called `Organization` should be renamed to `Org` for consistency.

I know it was subtle, but what do you think about also shortening "CustomizationTemplate" down to "CustomTemplate"?

Reply:
Sure, I'll get that changed as well.